src/separator.rs

```diff
@@ -20,6 +20,7 @@ use druid::{theme, Color, KeyOrValue};
 
 /// A separator widget.
 pub struct Separator {
+    size: KeyOrValue<f64>,
     width: KeyOrValue<f64>,
```

Comment:
That looks fishy, why add an unused field?

Reply:
Probably a bad merge. Thanks for pointing this out, I'll clean it up.